artifact-lang.sh

```diff
@@ -10,8 +10,7 @@ function retrieveLanguageParameters() {
   OWNER=${GIT_OWNER:-mdblp}
   REPO=translations
   # GIT_BRANCH can be a branch or a tag
-  # GIT_BRANCH=master
-  GIT_BRANCH=dblp.0.5.0
+  GIT_BRANCH=master
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏is it the branch main of the target repo ?

Reply:
Yes